src/compute/src/server.rs

```diff
@@ -487,6 +487,10 @@ impl<'w, A: Allocate + 'static> Worker<'w, A> {
                 compute_state.process_copy_tos();
             }
 
+            if let Some(compute_state) = &mut self.compute_state {
```

Comment:
```suggestion
```

Reply:
This doesn't work because in the first if-let, `compute_state` is an `ActiveComputeState`, while we need a `ComputeState` here.

We could change this to:
```rust
            if let Some(mut compute_state) = self.activate_compute(&mut response_tx) {
                compute_state.process_peeks();
                compute_state.process_subscribes();
                compute_state.process_copy_tos();
                compute_state.compute_state.process_sequential_hydration();
            }
```

Might be clearer, idk. I wrote the current code because it is consistent with how we do trace maintenance further up.

Reply:
Oh, never mind then!